keboliu/sonic-buildimage
# [Mellanox]New sku msn4700 new buffer size

**- What I did**
Update buffer size for 4700

**- How I did it**

**- How to verify it**
Make sure the system works normally with the new buffer size.
Run QoS test.
[new-buffer-test.4700.txt](https://github.com/keboliu/sonic-buildimage/files/4297983/new-buffer-test.4700.txt)

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
